shared/chat/conversations-list/index.desktop.js

```diff
@@ -1,62 +1,130 @@
-        <Avatar
-          size={32}
-          backgroundColor={globalColors.darkBlue4}
-          username={participants.first().username}
-          borderColor={conversation.get('unreadCount') ? globalColors.orange : undefined}
-        />
-        <Box style={{...globalStyles.flexBoxColumn, flex: 1, marginLeft: 12, position: 'relative'}}>
-          <Box style={{...globalStyles.flexBoxColumn, position: 'absolute', top: 0, bottom: 0, left: 0, right: 0}}>
-            <Usernames inline={true} type='Body' backgroundMode='Terminal' users={participants.toArray()} title={participants.map(p => p.username).join(', ')} />
-            <Text backgroundMode='Terminal' type='BodySmall' style={noWrapStyle}>{conversation.get('snippet')}</Text>
-          </Box>
+      onClick={() => onSelectConversation(conversation.get('conversationIDKey'))}
+      title={`${conversation.get('unreadCount')} unread`}
+      style={{...rowContainerStyle,
+        backgroundColor: isSelected ? globalColors.darkBlue2 : globalColors.transparent}}>
+        <Box style={{...globalStyles.flexBoxRow, flex: 1, maxWidth: 48, alignItems: 'center', justifyContent: 'flex-start', paddingLeft: 4}}>
+          <MultiAvatar singleSize={32} multiSize={24} avatarProps={avatarProps} />
+          {isMuted && <Icon type='iconfont-shh' style={shhStyle} />}
         </Box>
-        <Text backgroundMode='Terminal' type='BodySmall' style={{marginRight: 4}}>{formatTimeForConversationList(conversation.get('time'), nowOverride)}</Text>
-      </Box>)
-    })}
+        <Box style={{...globalStyles.flexBoxRow, flex: 1, borderBottom: `solid 1px ${globalColors.black_10}`, paddingRight: 8, paddingTop: 4, paddingBottom: 4}}>
+          <Box style={{...globalStyles.flexBoxColumn, flex: 1, position: 'relative'}}>
+            <Box style={{...globalStyles.flexBoxColumn, position: 'absolute', top: 0, bottom: 0, left: 0, right: 0, alignItems: 'center', justifyContent: 'center'}}>
+              <Usernames
+                inline={true}
+                type='BodySemibold'
+                style={{color: isMuted ? globalColors.blue3_40 : globalColors.white}}
+                containerStyle={{color: isMuted ? globalColors.blue3_40 : globalColors.white, paddingRight: 7}}
+                users={participants.toArray()}
+                title={participants.map(p => p.username).join(', ')} />
+              {snippet && !isMuted && <Text type='BodySmall' style={{...noWrapStyle, color: subColor}}>{snippet}</Text>}
+            </Box>
+          </Box>
+          <Text type='BodySmall' style={{marginRight: 4, alignSelf: isMuted ? 'center' : 'flex-start', color: subColor}}>{formatTimeForConversationList(conversation.get('time'), nowOverride)}</Text>
+      </Box>
+    </Box>
+  )
+}
+
+const shhStyle = {
+  color: globalColors.darkBlue2,
+  alignSelf: 'flex-end',
+  marginLeft: -5,
+  marginTop: 5,
+  // TODO remove this when we get the updated icon w/ the stroke
+  textShadow: `
```

Comment:
this was just to get it kinda looking close. I asked @cecileboucheron to give us a perfect icon instead of using the iconfont